src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -761,6 +768,12 @@ private async Task<List<LaunchCommand>> GetInitializeCommands()
                     if (!string.IsNullOrWhiteSpace(_launchOptions.WorkingDirectory))
                     {
                         string escapedDir = this.EnsureProperPathSeparators(_launchOptions.WorkingDirectory);
+
+                        if (IsCygwin)
+                        {
+                            escapedDir = CygwinFilePathMapper.MapWindowsToCygwin(escapedDir);
+                        }
```

Comment:
Should we just move this into `EnsureProperPathSeparators`? That seems to already be called in all the cases you are touching.

Reply:
Sorry I got confused because EscapeSymbolPath and EnsureProperPathSeparators are almost identical. But still, we might want to move your fix into those two functions (maybe getting rid of one of them)